src/Web/ViewModels/CustomAssessmentViewModel.cs

```diff
@@ -22,9 +22,7 @@ public class CustomAssessmentViewModel
 
     public enum EndpointVersion
     {
-        V1,
         V2,
```

Comment:
note that V2 is now listed first... making it the default when creating the request